### PR TITLE
Remove set legacy room in synapse

### PR DESCRIPTION
**Pull request by agusaldasoro**
As there was an error when updating the room when becoming friends, the legacy code to be compliant with old implementation was removed

### CODE REVIEW

**Review by Julieta11 — APPROVED** [review]

File: src/ws/service/friendship_event_updates.rs [hunk 1]
```diff
@@ -108,17 +107,6 @@ pub async fn handle_friendship_update(
     )
     .await?;
 
-    // Store the friendship event in the given room.
```
Comment (moliva) [inline src/ws/service/friendship_event_updates.rs:111]: This is no longer used since M1, right?

Comment (agusaldasoro) [reply src/ws/service/friendship_event_updates.rs:111]: Yes, that code was for retrocompatibility for friendships handled by Matrix. Can you confirm @Julieta11 ?

Comment (Julieta11) [reply src/ws/service/friendship_event_updates.rs:111]: In M1 we're still storing the event in the synapse room. This code was implemented for retrocompatibility to maintain the option to rollback to Matrix without losing any friendship interaction updates from M2 on.

Comment (moliva) [reply src/ws/service/friendship_event_updates.rs:111]: We won't be rolling back from M1 to Matrix for sure, if this code was meant for that, let's go ahead and remove it.

**Comment by agusaldasoro** [comment]
Closing this PR as there are other legacy actions that should be removed together, and the bug that was exposed here is fixed in https://github.com/decentraland/social-service/pull/272

**Pull request closed without merging**